frontend/app/routes/api.switch-language.ts

```diff
@@ -2,13 +2,14 @@ import { type ActionFunctionArgs, json } from '@remix-run/node';
 
 import { z } from 'zod';
 
+import { PREFFERRED_LANGUAGES } from '~/utils/constants';
 import { createLangCookie } from '~/utils/locale-utils.server';
 import { getLogger } from '~/utils/logging.server';
 
 const logger = getLogger('routes/api.switch-language');
 
 const switchLanguageDataSchema = z.object({
-  language: z.enum(['en', 'fr']),
+  language: z.enum(PREFFERRED_LANGUAGES),
```

Comment:
`PREFFERRED_LANGUAGES` language has nothing to do with the application UI languages. Of course it's the same values but they are not related. `PREFFERRED_LANGUAGES` is used for communication purposes. You changes can be reverted.

Reply:
I wanted to export the string literals into a constant so the code would be cleaner, but I see how the name doesn't fit. I can pick a different name.

Reply:
Remove it since UI languages is out of scope of this PR. `PREFERRED_LANGUAGES` can stay for now and add documentation to define the usage of that constant.

Reply:
Fixed: a001f766a197be2d079445840f63a22996af3319